### ISSUE TITLE
Cannot use custom dialogs with SPFx v 1.13

### ISSUE THREAD

**Issue by patrikhellgren** [SharePoint/sp-dev-docs#7431]
### Target SharePoint environment

SharePoint Online

### What SharePoint development model, framework, SDK or API is this about?

💥 SharePoint Framework

### Developer environment

Windows

### What browser(s) / client(s) have you tested

- [ ] 💥 Internet Explorer
- [ ] 💥 Microsoft Edge
- [ ] 💥 Google Chrome
- [ ] 💥 FireFox
- [ ] 💥 Safari
- [ ] mobile (iOS/iPadOS)
- [ ] mobile (Android)
- [X] not applicable
- [ ] other (enter in the "Additional environment details" area below)

### Additional environment details

- SPFx version 1.13
- Node.js version 14.16.1

### Describe the bug / error

I have upgraded a solution from 1.12.1 to 1.13 and after that the build fails with errors like the following for all custom dialogs:

```
error TS2339: Property 'show' does not exist on type...
error TS2339: Property 'close' does not exist on type...
error TS2339: Property 'domElement' does not exist on type...
```

VS Code does of course also show an error for these in the editor and it also warns for the import of sp-dialog that it "Could not find a declaration file for module '@microsoft/sp-dialog'". All other "@microsoft/sp-" imports looks ok.
I have also tested creating a new HelloWorld solution from scratch with the same result.

### Steps to reproduce

1.Use the following tutorial [Use custom dialog boxes with SharePoint Framework Extensions](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/extensions/guidance/using-custom-dialogs-with-spfx)
2.Check all use of this.close and this.domElement and you should see the problem.

### Expected behavior

Should be able to use custom dialogs with SPFx version 1.13.

**Comment by SamueleFurnari** [comment]
Hi, I have the same problem created with the v1.13. Following the suggestion, the solution buold but did not work. When running, the custom action button does not appears and the code crashes with an uncatchable error sayng:

```
Uncaught (in promise) Error
at t [as constructor] (listview-host-assembly_it-it_fd815997ef33cccdfbf8eac235f6f775.js:67)
at new t (listview-host-assembly_it-it_fd815997ef33cccdfbf8eac235f6f775.js:68)
at Function.e.buildErrorWithVerboseLog (listview-host-assembly_it-it_fd815997ef33cccdfbf8eac235f6f775.js:68)
at Function.e.buildLoadComponentError (listview-host-assembly_it-it_fd815997ef33cccdfbf8eac235f6f775.js:68)
at chunk.systemjs-component-loader_it-it_0e72155ab8ba4215d52d.js:1
```

I need to open a dialog that opens when the user select one or more file in a document library and allow user to select some information retrieved from external service, and than save the value returned from this service to a custom metadata.

If there is another way that allow me to do this, any suggestion is appreciated.

**Comment by patmill** [comment]
This should be fixed now with the 1.13.1 release.
